x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -147,26 +166,51 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
               data-test-subj={getTestId('field')}
             />
           </ConditionEntryCell>
-        </EuiFlexItem>
-        <EuiFlexItem>
+        </InputItem>
+        <InputItem style={{ gridArea: 'operator' }}>
           <ConditionEntryCell showLabel={showLabels} label={ENTRY_PROPERTY_TITLES.operator}>
-            <EuiFieldText name="operator" value={OPERATOR_TITLE.included} readOnly />
+            {entry.field === ConditionEntryField.PATH ? (
+              <EuiSuperSelect
+                options={operatorOptions}
+                onChange={handleOperatorUpdate}
+                valueOfSelected={entry.type}
+                data-test-subj={getTestId('operator')}
+              />
+            ) : (
+              <EuiFieldText
+                name="operator"
+                value={OPERATOR_TITLES.is}
+                data-test-subj={getTestId('operator')}
+                readOnly
+              />
+            )}
           </ConditionEntryCell>
-        </EuiFlexItem>
-        <EuiFlexItem grow={3}>
+        </InputItem>
+        <InputItem style={{ gridArea: 'value' }}>
           <ConditionEntryCell showLabel={showLabels} label={ENTRY_PROPERTY_TITLES.value}>
             <EuiFieldText
               name="value"
               value={entry.value}
+              placeholder={
```

Comment:
Super duper nit: can we move this to a util we can quickly unit test? It's a bit hard to read all the ternary operators.

Reply:
done 2dc4fd390cf5ea0e4fa67b3f5fc2561cbb29555e